packages/manager/modules/pci/src/projects/project/storages/objects/object/emptyUser/index.js

```diff
@@ -0,0 +1,27 @@
+import angular from 'angular';
+import '@ovh-ux/manager-core';
+import '@ovh-ux/ng-translate-async-loader';
+import '@uirouter/angularjs';
+import 'angular-translate';
+import '@ovh-ux/ui-kit';
+import 'ovh-api-services';
+
+import containers from '../../../containers';
+import routing from './emptyUser.routing';
+
+const moduleName = 'ovhManagerPciStoragesObjectsObjectEmptyUser';
+
+angular
+  .module(moduleName, [
+    containers,
+    'ngTranslateAsyncLoader',
+    'oui',
+    'ovh-api-services',
+    'ovhManagerCore',
+    'pascalprecht.translate',
+    'ui.router',
+  ])
+  .config(routing)
+  .run(/* @ngTranslationsInject:json ./translations */);
```

Comment:
Not required since we do not have translations here

Reply:
We do have translations for `emptyUser` modal